### PR TITLE
react-native-fluentui setup for android intial commit

### PR DESCRIPTION
**Pull request by mnkshukl**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [yes ] android

### Pull request checklist

This PR has considered (when applicable): NA
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Comment by Saadnajmi** [comment]
Why do we need to name the bundle and app name? Was that causing issues with running the app / solve your issues?

**Comment by mnkshukl** [comment]
> Why do we need to name the bundle and app name? Was that causing issues with running the app / solve your issues?

Mainly app.json and package.json bundle name should be same for making the bundle clubbed into assets.
Names changes are just part of that

File: apps/android/package.json
Comment (Saadnajmi) [inline apps/android/package.json:48]: I think our tooling is set up so that you can set `noPlatformSuffix` to false if you want to include the platform in the bundle.

**Review by Saadnajmi — CHANGES_REQUESTED** [review]

File: apps/android/package.json
Comment (mnkshukl) [reply apps/android/package.json]: it adds the suffix at last like index.bundle.android, so falling back to original naming convention

File: apps/android/package.json
Comment (Saadnajmi) [inline apps/android/package.json:16]: You don't need to include the arguments to the key. With just the key "start" you can run "yarn start --reset-cache", and it will pass the argument to the script (`react-native start`)

Comment (Saadnajmi) [reply apps/android/package.json:16]: Also, I think pakage.json doesn't like it when the key for a script has a space? Every other script I've seen has had no spaces

**Review by Saadnajmi — APPROVED** [review]

**Comment by Saadnajmi** [comment]
Not sure if you have merging privileges, but I can merge this if you don't @mnkshukl

**Comment by mnkshukl** [comment]
> Not sure if you have merging privileges, but I can merge this if you don't @mnkshukl

I don't have right now , Who can provide the merge access  ?

**Comment by Saadnajmi** [comment]
Probably @FalseLobster is the one to talk to, also @ejlayne . I'll merge this one

**Pull request merged**
